rdm/checklists/ISO_14971.txt

```diff
@@ -1,94 +1,36 @@
-description: 'WIP Audit checklist for ISO 14971 Medical devices - Application of risk management to medical devices'
-standard: "ISO 14971"
-version: "2007-10-01"
-requirements:
-  - rules:
-      - description: "1 Scope"
-        refs: []
-  - rules:
-      - description: "2 Terms and definitions"
-        refs: []
-  - rules:
-      - description: "3 General requirements for risk management"
-        refs: []
-  - rules:
-      - description: "3.1 Risk management process"
-        refs: []
-  - rules:
-      - description: "3.2 Management responsibilities"
-        refs: []
-  - rules:
-      - description: "3.3 Qualification of personnel"
-        refs:
-        - '14971:3.3'
-  - rules:
-      - description: "3.4 Risk management plan"
-        refs:
-        - '14971:3.4.a'
-        - '14971:3.4.b'
-        - '14971:3.4.c'
-        - '14971:3.4.d'
-        - '14971:3.4.e'
-#        - '14971:3.4.f'
-  - rules:
-      - description: "3.5 Risk management file"
-        refs: []
-  - rules:
-      - description: "4 Risk analysis"
-        refs: []
-  - rules:
-      - description: "4.1 Risk analysis process"
-        refs: []
-  - rules:
-      - description: "4.2 Intended use and identification of characteristics related to the safety of the medical device"
-        refs: []
-  - rules:
-      - description: "4.3 Identification of hazards"
-        refs:
-          - '14971:4.3'
-  - rules:
-      - description: "4.4 Estimation of the risk(s) for each hazardous situation"
-        refs: []
-  - rules:
-      - description: "5 Risk evaluation"
-        refs:
-          - '14971:5'
-  - rules:
-      - description: "6 Risk control"
-        refs: []
-  - rules:
-      - description: "6.1 Risk reduction"
-        refs:
-          - '14971:6.1'
-  - rules:
-      - description: "6.2 Risk control option analysis"
-        refs:
-          - '14971:6.2'
-  - rules:
-      - description: "6.3 Implementation of risk control measure(s)"
-        refs: []
-  - rules:
-      - description: "6.4 Residual risk evaluation"
-        refs: []
-  - rules:
-      - description: "6.5 Risk/benefit analysis"
-        refs: []
-  - rules:
-      - description: "6.6 Risks arising from risk control measures"
-        refs:
-          - '14971:6.6'
-  - rules:
-      - description: "6.7 Completeness of risk control"
-        refs: []
-  - rules:
-      - description: "7 Evaluation of overall residual risk acceptability"
-        refs:
-          - '14971:5'
-  - rules:
-      - description: "8 Risk management report"
-        refs:
-          - '14971:8'
-  - rules:
-      - description: "9 Production and post-production information"
-        refs:
-          - '14971:9'
+# Audit checklist for ISO 14971 Medical devices - Application of risk management to medical devices
+# Based on version 2007-10-01
```

Comment:
I think there is a 2019 version that came out. Perhaps we should name this file `14971_2007` for consistency, instead of `ISO_14971`? It may also be interesting to see what changed in the 2019 version (if we don't have a copy of the 2019 version, I can buy one)

Reply:
Good points. Yes, need the 2019 version.

Reply:
What version of 14971 should be included?

Reply:
The older version; the 62304 standard versions will mention the precise 14972 version that it requires . Perhaps the 2020 623049 will upgrade to the 2019 14971

Reply:
BTW I was mistaken about this; it doesn't seem that 62304 specifies a particular version.